CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ option(BUILD_THIRDPARTY "Wheter to build third party librarie or use preinstalle
 option(CODECOVERAGE "Enable Code Coverage Metrics in Clang" OFF)
 option(ENABLE_RESTART_RECOVERY_TESTS "Enable tests for restart recovery" OFF)
 option(BUILD_UTT "Build UTT library" OFF)
+option(CONCORD_ENABLE_ALL_METRICS "Enable all metrics in concord" ON)
```

Comment:
How is this option reflects in code?

Reply:
This option has to be removed as it is redundant. It was added when I was exploring ways to add compile time flags. I went with adding the flag in Makefile in this commit.

I need to evaluate if there are other ways to build concord, in which case I **might** need this variable.